core-java-modules/core-java-datetime-string/src/test/java/com/baeldung/formatinstant/FormatInstantUnitTest.java

```diff
@@ -0,0 +1,58 @@
+package com.baeldung.formatinstant;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.time.Instant;
+import java.time.ZoneId;
+import java.time.format.DateTimeFormatter;
+import java.time.temporal.UnsupportedTemporalTypeException;
+
+import org.joda.time.format.DateTimeFormat;
+import org.junit.Test;
+
+public class FormatInstantUnitTest {
+    
+    private static final String PATTERN_FORMAT = "dd.MM.yyyy";
+
+    @Test
+    public void givenInstant_WhenUsingDateTimeFormatter_thenFormat() {
```

Comment:
Lowercase the "w" in "when" for all the test method names

Reply:
done